core/domain/acl_decorators.py

```diff
@@ -68,6 +76,39 @@ def check_activity_accessible(
             activity_rights.viewable_if_private)
 
 
+def check_exploration_editable(user_id, user_actions, exploration_rights):
+    """Returns a boolean to signify whether given exploration is editable
+    by the user or not.
+
+    Args:
+        user_id: str. Id of the given user.
+        user_actions: list(str). List of actions given user can perform.
+        exploration_rights: Exploration rights object.
```

Comment:
on a second note. The _get_exploration_rights_ method raises _internal server error_ (in strict=True which is default) if exploration rights is None. Wouldn't it be better if it raises pageNotFoundException, that way there would be no need to check if exploration rights is None or not.